frontend/pages/ManageControlsPage/OSSettings/cards/Certificates/components/ViewCertificateModal/_styles.scss

```
@@ -0,0 +1,16 @@
+    display: flex;
+    flex-direction: column;
+    gap: $pad-large;
+  }
+
+  &__summary {
+    display: flex;
+    gap: $pad-xxlarge;
+  }
+
+  textarea {
+    font-family: "SourceCodePro", $monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SourceCodePro" (font-family-name-quotes)

(font-family-name-quotes)